CONTRIBUTING.md

```diff
@@ -74,27 +74,14 @@ This defaults to both.
 1. Push the tag: `git push vx.y.z` (or `git push origin vx.y.z` if you are working
    from your local machine).
 
-### Create a GitHub pre-release
+### Make the release public
 **Note:** Until the stable quality exercises have completed, the GitHub release
 should be officially marked as a `pre-release` (eg "non-production ready")
 
-1. From the Jenkins pipeline for the tag, retrieve the archived `dist/goreleaser`
-   directory.
-1. Create a GitHub release from the tag, add a description by copying the CHANGELOG entries
-   from the version, and upload the release artifacts from `dist/goreleaser`
-   to the GitHub release. The following artifacts should be uploaded to the release:
-   - CHANGELOG.md
-   - NOTICES.txt
-   - LICENSE
-   - summon-conjur-darwin-amd64.tar.gz
-   - summon-conjur-darwin-arm64.tar.gz
-   - summon-conjur-linux-amd64.tar.gz
-   - summon-conjur-windows-amd64.zip
-   - summon-conjur-solaris-amd64.tar.gz
-   - summon-conjur.deb
-   - summon-conjur.rpm
-   - SHA256SUMS.txt
-
+1. The tagged commit should have caused a Draft release to be created in GitHub.
```

Comment:
Should this say Pre-release instead of Draft release?

Reply:
I don't think so. Draft releases are not yet public. I believe you can also make a Draft public but mark it as a "pre-release" to imply that it's not fully baked yet. That's what the little comment at the top of the section that I left in is referring to. Should be Draft here, though.